app/reports/page-content.tsx

```diff
@@ -79,8 +79,12 @@ const buildPageTitle = ({
     additionalFragments.push('Include Muted')
   }
 
-  if (appealed) {
-    additionalFragments.push('Appealed')
+  if (appealed !== null) {
+    if (appealed === 'true') {
+      additionalFragments.push('Only Appeals')
+    } else {
+      additionalFragments.push('No Appeals')
+    }
   }
```

Comment:
Just lining the logic up with similar logic I see below.
```suggestion
  if (appealed === 'true') {
    additionalFragments.push('Only Appeals')
  } else if (appealed === 'false') {
    additionalFragments.push('No Appeals')
  }
```